pybikes/whitebikes.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class WhiteBikes(BikeShareSystem):
+
+    meta = {
+        'system': 'WhiteBikes'
+    }
+
+    def __init__(self, tag, meta, feed_url, bounding_box=None):
```

Comment:
This system, so far, does not use bounding boxes so there's no need to include it as an init parameter.

Reply:
True, bad copy n paste!